Makefile

```diff
@@ -55,7 +55,7 @@ build: ## Builds all containers or the one specified (n=service name)
 
 up: ## Starts all containers or the one specified (n=service name)
 	@echo "$(P) Starting all containers..."
-	@docker-compose --env-file .env -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml --profile service --profile core --profile kafka --profile utility up -d $(n)
+	@docker-compose --env-file .env -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml --profile core --profile kafka up -d $(n)
```

Comment:
Don't want to start `service` and `utility` services with the default `up` command.  As they normally fail unless everything has properly initialized.  Better to just wait and manually call the `make service-up` command when it'll work.